4.0.0/stability

rart opened the pull request:
- Bitmap formatting preference
- Using "full word width" on state & action bitmap masks
- Make disabled state visible for the edit mode switch
- Add disabled tooltip to edit mode switch when item is locked
- Changing order of switch/case on item icon/text display components to show `locked` over `modified`
- Add tooltip and additional styling options to PublishingStatusButtonUI
- [#4735] Changing sites management card to move publishing status to the footer craftercms/craftercms#4735
- Consolidate item actions types on models/Item.ts
- Fix hidden unconstrainded types on utils/content due to usage of partials

rart committed:
4.0.0/stability
- Bitmap formatting preference

sumerjabri commented on ui/app/src/utils/constants.ts:
```diff
@@ -15,44 +15,48 @@
-export const STATE_TRANSLATION_UP_TO_DATE_MASK /*   */ = 0b1000000000;
-export const STATE_TRANSLATION_PENDING_MASK /*      */ = 0b10000000000;
+//                                                         1|9876543210
+export const STATE_NEW_MASK /*                      */ = 0b000000000001;
```
Please switch to the full word width (64 bits)

rart committed:
4.0.0/stability
- Using "full word width" on state & action bitmap masks
- Make disabled state visible for the edit mode switch
- Add disabled tooltip to edit mode switch when item is locked
- Changing order of switch/case on item icon/text display components to show `locked` over `modified`
- Add tooltip and additional styling options to PublishingStatusButtonUI
- [#4735] Changing sites management card to move publishing status to the footer craftercms/craftercms#4735
- Consolidate item actions types on models/Item.ts
- Fix hidden unconstrainded types on utils/content due to usage of partials
